src/stackwalk.c

```diff
@@ -84,8 +84,28 @@ size_t rec_backtrace(uintptr_t *data, size_t maxsize)
     return rec_backtrace_ctx(data, maxsize, &context);
 }
 
+int jl_num_frames(uintptr_t ip, int skipC)
```

Comment:
I mainly mean that this is an internal API and you don't need to estimate it at all.